install_files/ansible-base/roles/app/tasks/copy_tor_url_info_to_app_dir.yml

```diff
@@ -23,12 +23,16 @@
     mode: "0644"
     content: |
       {{ v2_onion_url_lookup_result.stdout|default('') }}
+  notify:
```

Comment:
Making a note to remove this logic as part of https://github.com/freedomofpress/securedrop/issues/5731 as is it outside of the tor hidden service role